Packages/manifest.json

```diff
@@ -7,7 +7,7 @@
     "com.unity.ide.visualstudio": "2.0.15",
     "com.unity.probuilder": "4.2.3",
     "com.unity.settings-manager": "1.0.3",
-    "com.unity.test-framework": "1.1.33",
+    "com.unity.test-framework": "1.4.3",
```

Comment:
Is this manifest only referenced in testing? If not, does it work well with Unity 2019 since Input System goes back to 2019?

Reply:
The test framework is only used when running the tests. Some of the tests are broken when running in 2019 using either version of that framework. 

1.4.3 is listed as supported from 2019.2 upwards.

Reply:
ok, thanks